django/views/decorators/common.py

```diff
@@ -0,0 +1,14 @@
+from functools import wraps
+
+
+def no_append_slash(view_func):
+    """
+    Mark a view function as excluded from CommonMiddleware's APPEND_SLASH
+    redirection.
+    """
+    # view_func.should_append_slash = False would also work, but decorators are
+    # nicer if they don't have side effects, so return a new function.
+    def wrapped_view(*args, **kwargs):
```

Comment:
Most examples I've seen would decorate this function with `@wraps(view_func)` then return `wrapped_view`?

Reply:
🤷‍♀️ You say _potato_, I say _potato_ (Please supply accents.) 😀